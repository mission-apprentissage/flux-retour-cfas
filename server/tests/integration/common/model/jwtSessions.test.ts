import jwtSessionsModel from "@/common/model/jwtSessions.model";

describe("Model JwtSession", () => {
  it("should have a valid JSON schema", () => {
    expect(jwtSessionsModel.schema).toMatchSnapshot();
  });
});
