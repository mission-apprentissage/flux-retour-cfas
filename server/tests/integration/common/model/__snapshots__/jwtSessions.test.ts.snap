// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Model JwtSession should have a valid JSON schema 1`] = `
{
  "additionalProperties": false,
  "bsonType": "object",
  "properties": {
    "_id": {
      "bsonType": "objectId",
    },
    "jwt": {
      "bsonType": "string",
      "description": "Session token",
    },
  },
  "required": [
    "jwt",
  ],
}
`;
